### DISCUSSION TITLE
Writing a scientific paper to gain more potential GDL users from biology field

### DISCUSSION THREAD

**Discussion by pjb7687** [gnudatalanguage/gdl#888] [Ideas]
From several years ago, I have started contributing to GDL since I did my master's in a biophysics laboratory that extensively uses IDL for data analysis. As far as I know, as of today, many different groups in biology are also using IDL:

- http://www.medbiophysics.uni-hd.de/research/processing.html
- https://www.biorxiv.org/content/10.1101/2020.11.25.393017v5.full
- https://www.biorxiv.org/content/10.1101/2020.09.11.293639v1.full
- https://www.biorxiv.org/content/10.1101/2020.09.04.281022v1.full
- More papers, search "Interactive Data Language" (including double-quotes) @ bioRxiv website
- Unfortunately, no papers found with the term "GNU Data Language" @ bioRxiv
 
Since GDL is aiming to release 1.0 soon, I believe writing an introduction paper in the biology field would gain potentially more userbase and contributions from new users. For this, we can publish a citable article to bioRxiv or Zenodo free of charge.

**Comment by pjb7687** [comment]
Maybe I can start drafting on Google docs and share the link to the ones who would like to contribute to the paper..?

**Comment by slayoo** [comment]
@pjb7687 Alain recently has suggested promoting the 1.0 release as a JOSS paper: https://joss.theoj.org/

**Reply by slayoo** [reply]
JOSS submission workflow is based on github, so we could draft the paper using existing github workflows (PR, user permissions, etc) on a separate branch

**Reply by pjb7687** [reply]
Thanks! I am thinking about promoting GDL specifically in the biology field. I know many groups use IDL, but they don't seem to have heard about GDL although there are also several conference papers describing GDL. JOSS is a good journal, but I doubt its readership from the biology field.

**Reply by slayoo** [reply]
JOSS policy (https://joss.readthedocs.io/en/latest/submitting.html) explicitly mentions that: "Authors are welcome to submit their papers to a preprint server (arXiv, bioRxiv, SocArXiv, PsyArXiv etc.) at any point before, during, or after the submission and review process." so we can simply promote the JOSS paper by depositing it on bioRxiv as well.

**Reply by pjb7687** [reply]
Wow, I didn't know that. That would be awesome! Then I think adding some sentences regarding usage of IDL in biology fields and submit the paper to bioRxiv would gain more readership, since bioRxiv automatically tweets the new submission and promote it.

**Comment by alaingdl** [comment]
For my experience, because such different fields are really far, it might be better to write two small papers
focused on each field, and the specificities in each field (libs., examples, data type ...)

And we have to clarify the status on HDF 4/5, netCDF & mapping to consider some "paper" for remote sensing/Earth domain.
(what is really working well, what is missing : good point for future interships)

**Reply by slayoo** [reply]
I vote for a single short JOSS paper highlighting astro, bio, geo and other domains of relevance, and then re-posting it on both arXiv's astro-ph listing as well as on bioRxiv, so that the news will spread through tweets, mailings, RSS, etc
Why? Just to keep things simple and doable in short time horizon.

**Reply by pjb7687** [reply]
I like your idea of writing a quick and short paper, but I actually think that a writing separate paper would still be helpful since IDL is mainly being used by benchworkers (i.e. experimental biologists), and JOSS is really far from the journals having a broad readership among them. But rather than writing a full paper, perhaps writing a benchmark paper is also quickly doable. Specifically, we can contact some groups using IDL, and just do some benchmarks of their in-house IDL codes, and put the result in the paper - it won't be super hard. It could be also helpful to improve GDL too.
